chart/Chart.yaml

```diff
@@ -6,7 +6,7 @@ type: application
 
 version: 0.7.0
 appVersion: 0.7.0
-kubeVersion: ">= 1.19.0 < 1.22.0"
+kubeVersion: ">= 1.19.0-0 < 1.22.0-0"
```

Comment:
Why `-0` is needed?

Reply:
These are just added to test is kubectl, Kubernetes server versions in CI are causing the issue. They are not required reverting back my changes.